manifest.json

```diff
@@ -4,49 +4,62 @@
   "packaging_format": 1,
   "description": {
     "en": "The personal, minimalist, super-fast, no-database delicious clone",
-    "fr": "Clone de delicious, rapide, simple et sans base de données"
+    "fr": "Clone de delicious, rapide, simple et sans base de données."
   },
   "version": "0.12.1~ynh2",
   "url": "https://github.com/shaarli/Shaarli",
-  "upstream": {
-        "license": "MIT",
-        "demo": "https://demo.shaarli.org/",
-        "admindoc": "https://shaarli.readthedocs.io/en/master/",
-        "code": "https://github.com/shaarli/Shaarli"
-    },
   "license": "MIT",
   "maintainer": {
     "name": "lapineige et rafi59",
     "email": "rafi59_dev@srvmaison.fr.nf",
     "url": ""
   },
   "requirements": {
-    "yunohost": ">= 4.3.0"
+    "yunohost": ">= 3.8.1"
```

Comment:
Ah in fact it's probably this: requiring yunohost higher than 4.2 implicitly triggers some permissions on /var/www/$app

We probably need to add some tweaks like `chown $app:www-data /var/www/$app` in the install script (and upgrade and restore) assuming it's not already there

Reply:
The thing is, according to this report https://github.com/YunoHost-Apps/shaarli_ynh/issues/75#issuecomment-986056415 the is no permission issue, they are identical…